components/board/analog_smoother.go

```diff
@@ -87,6 +91,10 @@ func (as *AnalogSmoother) Start(ctx context.Context) {
 	//    numSamples        4
 
 	numSamples := (as.SamplesPerSecond * as.AverageOverMillis) / 1000
```

Comment:
Just noticing AverageOverMillis could be zero as well.

Reply:
That should be handled fine by `if numSamples <= 0 {` though, no? We only ever divide by SamplesPerSecond, not AverageOverMillis

Reply:
Yeah but if we pass in a SamplesPerSecond with no time, I think that implies "average over the last 5 samples", regardless of time (which I guess we should default), but we're actually locking it to 1 right now. 

Either way - none of my comments are blocking (I just want that to be clear @penguinland), we can make follow-up tickets to improve the analog reader incrementally.